store/snackbar.ts

```diff
@@ -1,18 +1,20 @@
 import { ActionTree, MutationTree } from 'vuex';
 
+type colors = 'error' | 'warning';
+
 export interface SnackbarState {
   message: string;
-  color: string;
+  color: colors;
 }
 
 export const state = (): SnackbarState => ({
   message: '',
-  color: ''
+  color: 'error'
```

Comment:
Default should likely be left empty, to avoid accidentally showing error snackbars when wanting to inform the user of something benign.